project_e/jobs/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.urls import reverse
+#from hashid_field import HashidAutoField
+from project_e.customers.models import Customer
+from project_e.dealers.models import Dealer
+from project_e.contractors.models import Contractor
+
+#import customer as OneToOneField (1:1)
+#import dealer as foreign key (Many:1)
+
+
+class Job(models.Model):
+    job_id = models.AutoField(primary_key=True)
+    #cust_id = models.OneToOneField(Customer, blank=True, null=True, on_delete=models.CASCADE)
+    cust_id = models.IntegerField(default=0)
+    dealer_id = models.ForeignKey(Dealer, blank=True, null=True, on_delete=models.CASCADE)
+    #salesman_id = models.IntegerField()
+    cont_id = models.ForeignKey(Contractor, blank=True, null=True, on_delete=models.CASCADE)
+    sale_date = models.DateTimeField(_("Date of Sale"), blank=True, null=True)
+    contractor_notified_date = models.DateTimeField(_("Date Given To Contractor"), blank=True, null=True)
+    contractor_completed_date = models.DateTimeField(_("Date Completed by Contractor"), blank=True, null=True)
+    pref_inst_day1 = models.DateTimeField(_("Preferred Installation Time 1"), blank=True, null=True)
```

Comment:
It may be better to abstract the installation into another field, but lets leave these here for now